packages/transformers/elm/package.json

```diff
@@ -29,6 +29,9 @@
     "nullthrows": "^1.1.1",
     "terser": "^5.14.2"
   },
+  "devDependencies": {
+    "@parcel/types": "2.7.0"
+  },
   "peerDependencies": {
     "elm": "^0.19.1-5"
```

Comment:
Worth noting that `elm` does not work on `linux-arm64` meaning this solution cannot be built on docker for MacOS (or ARM Linux, Asahi, etc)